documentation/slate/source/includes/disruptions.md

```diff
@@ -0,0 +1,60 @@
+Disruptions
+===========
+
+
+In this paragraph, we will explain how the disruptions are displayed in the different APIs.
+
+# Context
+
+## periods
+
+### production period of coverage data
+The production period is global to a navitia coverage and is the validity period of the coverage's data.
+This production period cannot exceed one year.
+
+### publication period of a disruption
+The publication period of a disruption is the period on which we want to display the disruption in navitia.
+
+The creator of the disruption might not want the traveller to know of a disruption before a certain date (because it's too uncertain, secret, ...)
+The publication period is the way to control this.
+
+### application periods of a disruption
+The application periods are the list of periods on which the disruption is active.
+
+## query dates
+
+Another thing is that can be tricky to understand this that there can be several datetimes for a query.
+
+### now
+
+In this document 'now' will represent the actual datetime at which the query is made, so the current datetime.
+
+# Summary
+
+To sum up we display an impact if 'now' is in the publication period.
+
+The status of the impact depends only of 'now' and is:
+* 'active' if 'now' is inside an application period
+* 'future' if there is an application periods after 'now'
```

Comment:
period (no `s`)
and we still have no idea about the priority to apply these rules